src/components/HOCs/WithCurrentTask/WithCurrentTask.jsx

```diff
@@ -277,6 +277,7 @@ export const mapDispatchToProps = (dispatch, ownProps) => {
     },
 
     fetchOSMElementHistory,
+    fetchOSMElement,
```

Comment:
Is this the correct approach or should I pass this function in another way?

Reply:
That approach looks good to me.